packages/flutter_tools/lib/src/cache.dart

```diff
@@ -101,6 +101,16 @@ class Cache {
     return dir;
   }
 
+  File getDevFSLoaderScript() {
+    return new File(path.absolute(path.join(getRoot().path,
```

Comment:
I would look relative to `cache.flutterRoot`, and probably move this method into `run.dart`.